src/commands/kick.ts

```diff
@@ -29,13 +28,13 @@ botCache.commands.set(`kick`, {
   botServerPermissions: [
     "KICK_MEMBERS",
   ],
-  execute: (message, _args, guild) => {
-    const member: Member = _args["member"];
-    const reason: string = _args["reason"];
+  execute: async (message, args, guild) => {
+    const member: Member = args.member;
```

Comment:
This is fine and does work but it goes against the consistency with the rest of the commands. If possible can you do it similar to how the guide showed using something like

```ts
execute: async (message, args: KickArgs, guild) => {

}

interface KickArgs {
  member: Member;
  reason: string;
}
```
Once you do that both these lines can be removed